### ISSUE TITLE
Create an illustration of how WebRTC Star network is set up (Rendezvous point at the middle)

### ISSUE THREAD

**Issue by diasdavid** [libp2p/js-libp2p-webrtc-star#117]

**Comment by Ivshti** [comment]
@diasdavid can we have a bit more clarification on this?

1. As far as I can see,  the signalling server is only used to exchange mutladdr's, rather than a full ICE connectivity description - am I understand this right?
2. This would mean there is no TURN fallback, is that correct?

**Comment by diasdavid** [comment]
> As far as I can see, the signalling server is only used to exchange mutladdr's, rather than a full ICE connectivity description - am I understand this right?

No. The SS is really exchanging the SDP offers and uses multiaddrs to route them properly

> This would mean there is no TURN fallback, is that correct?

This is correct.
